Slim/Player/Squeezebox2.pm

```diff
@@ -296,15 +297,18 @@ sub volume {
 		my $preamp = 255 - int( 2 * ( $prefs->client($client)->get('preampVolumeControl') || 0 ) );
 
 		my $data;
+		my $balance = $prefs->client($client)->get('balance') || 0;
+		my $left = $balance > 0 ? (25 - $balance) / $balance : 1;
+		my $right = $balance < 0 ? (25 + $balance) / -$balance : 1;
```

Comment:
Don't we risk to pull one channel to eleven? Should we cap the channel volume, or does the player do this?

Reply:
I'm a bit confused here, why eleven? $balance is -25..25 and $left & $right are supposed to be ratio where one will be set to 1 and the other within [0..1[ that would be a multiplier to gain. Very basic...

Reply:
https://youtu.be/4xgx4k83zzc 😁 

We should have some `setValidate()` in place to make sure it doesn't go beyond the expected +/-25 (see line 57). The slider is only rendered in the default skin, but not in Classic and more importantly Material.